src/components/SummaryStats/SummaryStats.tsx

```diff
@@ -87,7 +69,7 @@ const SummaryStat = ({
         {value == null ? null : (
           <>
             <StatValueText condensed={condensed} isEmbed={isEmbed}>
-              {formatValueForChart(chartType, value)}
+              {formatValue(chartType, value, '')}
```

Comment:
The old code used `-` as the "null value".  Also, I'm a fan of inline comments for documenting literal values passed to functions, e.g.:

```
{formatValue(chartType, value, /*nullValueCopy=*/'-')}
```